pkg/querier/queryrange/limits.go

```diff
@@ -28,6 +28,10 @@ const (
 	limitErrTmpl = "maximum of series (%d) reached for a single query"
 )
 
+var (
+	ErrMaxQueryParalellism = fmt.Errorf("proto: max query parallelism")
```

Comment:
Not sure about this error message. The user will receive it.

Reply:
Any suggestions for alternatives?

Reply:
Something saying querying is disable and you should get in touch with you cluster operator